.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'  # Trigger on version tags
+  schedule:
+    - cron: '0 0 * * *'  # Schedule cron
```

Comment:
I think you're mixing two incompatible suggestions here. @dlebauer was suggesting a cron job to keep build artifacts (which are temporary) from expiring, I was suggesting release and testing should be separate workflows that each compile their own binary and therefore do not need to create build artifacts in the first place.

I'll make some other suggestions below to show what I meant.